Sources/RichTextKit/Context/RichTextContext.swift

```diff
@@ -55,114 +55,91 @@ public class RichTextContext: ObservableObject {
     public internal(set) var selectedRange = NSRange()
 
 
-    // MARK: - Observable properies
+    // MARK: - Public properies
 
+    /// Whether or not the text is currently being edited.
+    @Published
+    public var isEditingText = false
+    
+    /// The current text alignment, if any.
+    @Published
+    public var textAlignment: RichTextAlignment = .left
+    
+    /// The current font name.
+    @Published
+    public var fontName = ""
+    
+    /// The current font size.
+    @Published
+    public var fontSize = CGFloat.standardRichTextFontSize
+    
+    /// The currently highlighted range, if any.
+    public var highlightedRange: NSRange?
+    
+    /// Use this Publisher to emit any attribute changes to textView.
+    public let userActionPublisher: PassthroughSubject<RichTextAction, Never> = .init()
+    
+    // MARK: - Internal properties
+    
     /// The current background color, if any.
     @Published
-    public var backgroundColor: ColorRepresentable?
+    var backgroundColor: ColorRepresentable?
 
     /// Whether or not the current rich text can be copied.
     @Published
-    public var canCopy = false
+    var canCopy = false
 
     /// Whether or not the latest undone change can be redone.
     @Published
-    public var canRedoLatestChange = false
+    var canRedoLatestChange = false
```

Comment:
But actually, removing `public` from these are breaking changes. I think we can do it if we don't release this PR as a 0.9 patch, but as a 1.0 bump.